pg_nearest_city/scripts/voronoi_generator.py

```diff
@@ -11,19 +11,68 @@
 import gzip
 import logging
 import os
+import re
 import shutil
+import subprocess
 import tempfile
 import urllib.request
 import zipfile
-from dataclasses import dataclass
+from collections import ChainMap
+from dataclasses import dataclass, field
+from datetime import datetime, timedelta
 from pathlib import Path
 from typing import Optional
 
 import psycopg
+from pg_nearest_city.db.data_cleanup import ROWS_TO_CLEAN, make_queries
 from pg_nearest_city.db.tables import get_tables_in_creation_order
 from psycopg.rows import dict_row
 
 
+@dataclass
+class Header:
+    """Class to create a k:v pair instead of dataclasses' asdict method."""
+
+    key: str
+    value: str
+
+    def _to_dict(self):
+        return {self.key: self.value}
+
+
+@dataclass
+class URLConfig:
+    """Class representing a data file.
+
+    domain: the domain portion of the URL (i.e. after http[s]:// and before the first /)
+    path: the path portion of the URL (i.e. everything after the domain)
+    alpha3_column: the name of the column in the file with an ISO 3166-1 alpha-3 code
```

Comment:
This is quite specific, unlike the rest. Perhaps this should be refactored to either a base class / inherited class, or some other means.